package/rpm/SOURCES/usr/sth/conf/sth_default.conf

```diff
@@ -0,0 +1,71 @@
+#!/usr/bin/env bash
+
+# Short Term History configuration variables.
+# STH is configured defining the folowing variables. 
+# All variables are optional, so, if one of this variables is not defined 
+# default value is used. 
+
+# See https://github.com/telefonicaid/IoT-STH#running-the-sth-server 
+# for further information
+
+# Note: is important to export environment variables with export
+
+# The host where the STH server will be started. Default value: "localhost".
+export STH_HOST="localhost"
```

Comment:
I think that if it is the HOST BIND better use 0.0.0.0, because is possible to access remotely and not  local only.